docs/chapter7/Agent/demo.py

```
@@ -5,9 +5,10 @@
 
 
 if __name__ == "__main__":
+    QWEN_API_KEY = os.getenv("QWEN_API_KEY")
```

Ruff (0.12.2)

Undefined name `os`

(F821)